### ISSUE TITLE
Unable to Connect To Shared-In Resources As Admin Of External Tailnet; [Out:GitHub;In:Microsoft]

### ISSUE THREAD

**Issue by TumM0reNcE5SsIti** [tailscale/tailscale#11616]
### What is the issue?

Unable to access shared-in resources from GitHub>Microsoft
Issues "started today" without any changes (perceived).

### Steps to reproduce

- Connect to Tailscale via microsoft connected machine
- Attempt to access shared-in machine (known working)
- Unable to connect to any resources

### Are there any recent changes that introduced the issue?

None perceived.

### OS

Windows

### OS version

Windows 11

### Tailscale version

1.62.1 (t8ee5801a3-g76428ac0b)

**Comment by TumM0reNcE5SsIti** [comment]
Whoever is looking into this, I think I found out what broke as far as re-adding the device to the external tailnet.
- Login to login.tailscale.com/admin as admin of a tailscale (not owner)
  - Switch to tailscale you are adding the shared machine to
- Use shared link to add machine to tailscale
  - Accept the invite (but it registers the machine with the logged in user; not the tailscale of the customer being "admin"d for)
- In the portal for the person adding it on their behalf, the machine shows up as added (causing a LOT of confusion)
- Customer can't login and see the machine added to their tailscale (something is broken)
- Connectivity can't be made, simply because the connection doesn't exist

In short, I admin for a half dozen people, and I can't "re-add" connections to my tailscale through my own portal, because something is inherently broken that won't allow me to share a link, and give external users access, because when I do the above, **they can't see the machine in their portal**.

**Comment by TumM0reNcE5SsIti** [comment]
After "sharing correctly", I as the admin, cannot see the shared machine in my portal, as an admin for their tailnet.
Something is broken, and it's causing confusion and a lot of headaches.

**Comment by TumM0reNcE5SsIti** [comment]
It has been three days without triage and I'd love to know where this aligns with the Tailscale team as far as priority, please.

**Comment by kelivel** [comment]
Hi @TumM0reNcE5SsIti, I'm not sure that I fully understand the situation and would like to take a closer look at your tailnet. Can you reach out to our support team at http://www.tailscale.com/support and let us know the exact device(s) shared, to which tailnet, and who should have access? Thank you!

**Comment by TumM0reNcE5SsIti** [comment]
@kelivel I've submitted the support request and have received an email confirming creation.

**Comment by TumM0reNcE5SsIti** [comment]
Also, **Very Few** may be **impacted**, but I believe every Tailscale user is affected by this process breakage. I don't believe it to be possible as a single user / single tenant problem.

That's just my two cents, though.

**Comment by TumM0reNcE5SsIti** [comment]
This issue has resurfaced, with a vengeance, as now the **correct** way to share a node is broken as well.
